### ISSUE TITLE
error on core.py file

### ISSUE THREAD

**Issue by LlewopNomis** [zostera/django-bootstrap5#229]
Hi, Just a tiny error on the core.py file that I forgot to fix on loading the app in a new environment, the final 's' is missing on line 13:
```
        "url": "https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.j",
```
Rather than:
```
        "url": "https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js",
```
Easy enough to fix but thought it might be handy to prevent any panic moments!

Kind Regards,

Simon

**Comment by LlewopNomis** [comment]
I've got a production system that I can't go live with due to this issue on pythonanywhere; whilst I can fix it in my local virtualenv, I can't find the directory for the virtualenv on pythonanywhere, could you take a look and fix this so I can reinstall the package once it's fixed?

**Comment by akx** [comment]
This seems to be fixed in 21.2+.

**Comment by dyve** [comment]
👍
